src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -207,8 +214,10 @@ class PillSelector extends React.Component<Props, State> {
                         onBlur={this.resetSelectedIndex}
                         ref={this.hiddenRef}
                         tabIndex={-1}
+                        data-testid="pill-selection-helper"
```

Comment:
If this attribute is used for the sole purpose for finding in tests, can you use some other attribute on this span?

Reply:
Actually I think we've been moving _toward_ adding these test IDs in favor of more brittle selectors like CSS classes or other various attributes. This is also the Cypress recommendation as well: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

The previous class based brittle selector is why the current tests gave a false positive. Any particular reason to avoid adding a `data-testid`?

Reply:
It's fine, it will get stripped out in the release builds